c10/core/DispatchKey.cpp

```diff
@@ -96,6 +96,9 @@ const char* toString(DispatchKey t) {
     case DispatchKey::VmapMode:
       return "VmapMode";
 
+    case DispatchKey::Math:
+      return "Math";
```

Comment:
I don't know if you wanna do it in this PR, but it might be time to start using macros [a la ScalarType.h](https://github.com/pytorch/pytorch/blob/master/c10/core/ScalarType.h#L25-L41) to define these sorts of functions...

Reply:
tracked in https://github.com/pytorch/pytorch/issues/44657 :D